### DISCUSSION TITLE
Not ready for a new version yet?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/thorium#98] [General]
M111.0 has been released for several days, are you planning to skip 110.0 this time? The fcdm1.com video, the problem of the sound of the sound, does not exist in the Android version of Google Canary 111.0. It is worth thinking deeply and formulating a solution. Thanks!

**Comment by gz83** [comment]
> M111.0 has been released for several days, are you planning to skip 110.0 this time? The fcdm1.com video, the problem of the sound of the sound, does not exist in the Android version of Google Canary 111.0. It is worth thinking deeply and formulating a solution. Thanks!

@whxuezhe Please be patient and wait for the new version to be released
